src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/VectorConfig.java

```diff
@@ -51,17 +81,8 @@ public static VectorConfig fromJson(JsonNode jsonNode, ObjectMapper objectMapper
           vectorizeServiceParameterNode == null
               ? null
```

Comment:
Could use `Map.of()` unless we want to use `null` as marker of some kind.

Reply:
Didn't understand this recommendation.